src/coreclr/jit/importer.cpp

```diff
@@ -9630,9 +9657,12 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
         }
 #endif // DEBUG
 
+        bool isIntrinsic = (mflags & CORINFO_FLG_INTRINSIC) != 0;
```

Comment:
```suggestion
        const bool isIntrinsic = (mflags & CORINFO_FLG_INTRINSIC) != 0;
```
Since this is just a cosmetic change, you can also add const to make it obvious that it is not changed anywhere.